public/data/aparna2071.json

```
@@ -0,0 +1,87 @@
+{
+    "name": "Aparna Soneja",
+    "type": "personal",
+    "bio": "Open Source Enthusiast | MERN Stack Web developer at Accenture",
+    "avatar": "https://github.com/aparna2071.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/aparna2071",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/AparnaSoneja",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/aparna-35066b191/",
+        "icon": "linkedin"
+      },
+      {
+        
+        "name": "Read my Blog",
+        "url": "https://chattingappusingreact.blogspot.com/",
+        "icon": "book"
+          
+      }
+    ],
+    "milestones": [
+        {
+            "title": "Open source contributor and Member of EddieHub community",
+            "date": "March 2022-Present",
+            "icon": "github",
+            "description": "Contributing to various open source projects, joined EddieHub as a Member on GitHub and as a helper on Discord",
+            "url": "https://github.com/EddieHubCommunity"
+          },
+      {
+        "title": "MERN Stack developer at Accenture",
+        "date": "August 2021-Present",
+        "icon": "dollar",
+        "description": "Started job as a Full stack web developer and got recognised as a Merit holder in MERN stack training",
+        "url": "https://www.accenture.com/in-en"
+      },
+      {
+        "title": "Graduated from GGSIPU, India with a CGPA of 9.12",
+        "date": "July 2021",
+        "icon": "Graduation Hat",
```

I believe this should have a hyphen. 👀
```suggestion
        "icon": "graduation-hat",
```

---

Well spotted, yes this should be as @Panquesito7 suggested, otherwise it won't show up